open_seq2seq/decoders/rnn_decoders.py

```diff
@@ -30,8 +30,10 @@ def get_required_params():
       'tgt_emb_size': int,
       'attention_layer_size': int,
       'attention_type': ['bahdanau', 'luong', 'gnmt', 'gnmt_v2'],
-      'decoder_cell_units': int,
-      'decoder_cell_type': ['lstm', 'gru', 'glstm', 'slstm'],
+      #'decoder_cell_units': int,
+      #'decoder_cell_type': ['lstm', 'gru', 'glstm', 'slstm'],
+      'core_cell': None,
+      'core_cell_params': dict,
```

Comment:
Should this go into optional_params since it defaults to empty dictionary if not probided?